docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Application support policy
+---
+
+A defined application support policy allows Great Expectations (GX) to better prioritize its response to application integration, operating system, and programming language issues. Formally defining the support policy also provides contributors with a better understanding of who is responsible for resolving application integration, operating system, and programming language issues.
+
+## Support categories
+
+The following are the types of support available for application integration, operating system, and programming language issues:
+
+- **GX supported** - The GX Developer Relations (DevRel) team triages, reproduces, and then assigns issues to the appropriate GX Engineering team. Issues with application integration, operating system, and programming languages in maintenance mode are reviewed, but new features or functionality are not implemented as part of the resolution.
+
+- **Community supported** - The GX DevRel team triages and reproduces issues, and then determines if GX should implement new features or functionality to resolve the issue. If the DevRel team determines that new features or functionality are not warranted, the GX community is responsible for issue resolution. If new features or functionality are required, DevRel assigns the issue to the appropriate GX Engineering team.
+
+## GX supported
+
+The following are the levels of support offered by GX for integrated applications, operating systems, and programming languages.
+
+### Operating systems
+
+The following table defines the operating systems supported by GX Cloud and GX OSS.
+
+| GX Cloud                     | GX OSS                        |
+|------------------------------|-------------------------------|
+| - Mac/Linux ¹                | - Mac/Linux                   | 
```

Comment:
I'd like to remove these bullets for single items, I think they make things harder to read,